client/src/components/scoreStepsList.test.tsx

```diff
@@ -3,6 +3,12 @@ import {render} from '@testing-library/react';
 import ScoreStepsList from './scoreStepsList';
 import {LocalizedComponent} from '../test/testHelpers';
 
+beforeAll(() => {
```

Comment:
Can we add the following to right above line 6?

// TODO: Move this to a location that will detect on all tests
// See ticket: https://github.com/usds/justice40-tool/issues/550

Reply:
You should try out the "code suggestion" github feature! 
![image](https://user-images.githubusercontent.com/74203452/129941739-1381fd60-6ede-495f-b185-1240c949aa51.png)